### PR TITLE
Update port forwarding docs for reverse portfwd

### PR DESCRIPTION
**Pull request by gememma**

### CODE REVIEW

**Comment by gememma** [comment]
Latest commit: 	d058d99
Status:	 ✅  Deploy successful!
Preview URL:	https://99ceec53.mirrord-dev.pages.dev/
Branch Preview URL:	https://gemma-mbe-443-documentation.mirrord-dev.pages.dev/

File: content/docs/using-mirrord/port-forwarding.md [hunk 1]
```diff
@@ -19,16 +19,21 @@ You can use the command like so:
 mirrord port-forward --target <target-path> -L <local port>:<remote address>:<remote port>
 ```
 
-For example, to forward traffic from localhost:8080 to py-serv on port 80 in targetless mode:
+For example, to forward traffic from localhost:8080 to py-serv on port 80 (in targetless mode):
```
Comment (Razz4780) [inline content/docs/using-mirrord/port-forwarding.md:22]:
```suggestion
For example, to forward traffic from localhost:8080 to an incluster service py-serv listening on port 80:
```

File: content/docs/using-mirrord/port-forwarding.md [hunk 2]
```diff
@@ -19,16 +19,21 @@ You can use the command like so:
...
 ```bash
 mirrord port-forward -L 8080:py-serv:80
 ```
 
+It also allows for reverse port forwarding, where traffic to a port on the target pod is forwarded to a local one, like so:
```
Comment (Razz4780) [inline content/docs/using-mirrord/port-forwarding.md:27]:
```suggestion
It also allows for reverse port forwarding, where traffic is redirected from a port on the target pod or workload to a local port, like so:
```

File: content/docs/using-mirrord/port-forwarding.md [hunk 3]
```diff
@@ -19,16 +19,21 @@ You can use the command like so:
...
+```bash
+mirrord port-forward --target <target-path> -R <remote port>:<local port>
+```
+
```
Comment (Razz4780) [inline content/docs/using-mirrord/port-forwarding.md:31]: I'd add another example, like:

For example, to forward traffic from an incluster deployment py-serv listening on port 80 to localhost:8080:
```bash
mirrord port-forward --target deployment/py-serv -R 80:8080
```

File: content/docs/using-mirrord/port-forwarding.md [hunk 4]
```diff
@@ -19,16 +19,21 @@ You can use the command like so:
...
+In addition, multiple ports can be forwarded in one direction or both directions simultaneously in the same command by providing each source and destination as a separate `-L` or `-R` argument.
```
Comment (Razz4780) [inline content/docs/using-mirrord/port-forwarding.md:32]:
```suggestion
In addition, multiple ports can be forwarded in one direction or both directions simultaneously in the same command by providing each source and destination as a separate `-L` or `-R` argument.

Regular port forwarding with an `-L` can be done in targetless mode and does not require specifying any target. Reverse port forwarding always requires a target.
```

File: content/docs/using-mirrord/port-forwarding.md [hunk 5]
```diff
@@ -19,16 +19,21 @@ You can use the command like so:
...
-- The local port part of the argument is optional, with out the same port will be used locally as on the remote.
+- The local port component of the `-L` argument is optional, and without it the same port will be used locally as on the remote.
+- The same is true of the `-R` argument: if one port number is provided, it will be used for both local and remote ports.
 - Port-forwarding only supports TCP, not UDP.
 - The remote address can be an IPv4 address or a hostname - hostnames are resolved in the cluster.
-- Connections are made lazily, so a hostname may fail to resolve only after mirrord attempts to send it data.
-
-In the future, mirrord will also support reverse port-forwarding ([tracked by this issue](https://github.com/metalbear-co/mirrord/issues/2609)).
\ No newline at end of file
+- Connections are made lazily, so a hostname may fail to resolve only after mirrord attempts to send it data.
```
Comment (Razz4780) [inline content/docs/using-mirrord/port-forwarding.md:39]:
```suggestion
- In regular port forwarding (`-L`) connections are made lazily and hostname resolution is attempted only data is sent to the local port.
```

**Review by Razz4780 — CHANGES_REQUESTED** [review]
A couple suggestions

File: content/docs/using-mirrord/port-forwarding.md (hunk 4 above)
Comment (gememma) [reply content/docs/using-mirrord/port-forwarding.md:32]: good catch 👀

**Review by Razz4780 — APPROVED** [review]

**Pull request merged**